tests/installation/zdup.pm

```diff
@@ -90,6 +90,7 @@ sub run() {
                        [ -z \$dev ] || echo \"found dev \$dev with label \$label\"";
             # if that fails, e.g. if volume descriptor too long, just try /dev/sr0
             $defaultrepo = "dvd:/?devices=\${dev:-/dev/sr0}";
+            $defaultrepo = "cd:/?devices=\${dev:-/dev/sr0}" if get_var('PATCH');
```

Comment:
Without this change zypper_lr is complaining about missing repo and test fail. Could this be related to the fact that this testcase will use BOOTFROM=d variable?